docs/DIRECTIVES.md

```
@@ -114,6 +117,84 @@ server {
 
 Normally `auth_oidc_mode off` is sufficient, but use `auth_oidc off` when you want to completely exclude OIDC module processing (e.g., static file serving).
 
+### auth_oidc_bearer
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)